### ISSUE TITLE
Error "llama_model_load: error loading model: illegal split file: <number>, model must be loaded with the first split"

### ISSUE THREAD

**Issue by felladrin** [ngxson/wllama#135]
While setting up v2.0, I've noticed it's not able to load [this model](https://huggingface.co/Felladrin/gguf-Q8_0-SmolLM2-135M-Instruct/tree/main):
```
hfRepoId: "Felladrin/gguf-Q8_0-SmolLM2-135M-Instruct",
hfFilePath: "model.shard-00001-of-00005.gguf",
```
[Actually, the same error is happening with all models that I split (which were all working on v1).]

It downloads the models correctly, but if triggers the following error when loading the model:
```
llama_model_load: error loading model: illegal split file: <number>, model must be loaded with the first split
```

<img width="1431" alt="image" src="https://github.com/user-attachments/assets/05a3d327-af22-49db-8342-be1b8167616c">

Any clues?

## Device info

OS: MacOS
Browser: Tested on Brave, Chromium, and Safari

## How to reproduce

- Access https://github.ngxson.com/wllama/examples/main/dist/
- Click the "+ Add GGUF" button
- Fill the modal with `Felladrin/gguf-Q8_0-SmolLM2-135M-Instruct` and `model.shard-00001-of-00005.gguf`, and save.
- Download the model
- Try to load the model, and it will inform the error.

**Comment by felladrin** [comment]
Update:

I've just noticed it's actually happening with any split GGUF model, including the ones from the [Demo](https://github.ngxson.com/wllama/examples/main/dist/).

I've tried loading the `qwen2-1_5b-instruct-q4_k_m-(shards).gguf` and got the same error:

<img width="632" alt="image" src="https://github.com/user-attachments/assets/7d1b1111-ad33-46b2-9641-060594255bde">

<img width="830" alt="image" src="https://github.com/user-attachments/assets/274b98eb-72c2-4f15-a488-5ca3b88d05cb">

**Comment by felladrin** [comment]
Update:
I've confirmed the issue in these 3 browsers (Brave, Chromium, and Safari), on MacOS, in the [Demo](https://github.ngxson.com/wllama/examples/main/dist/) app.

**Comment by ngxson** [comment]
Should be fixed in v2.0.1

**Comment by felladrin** [comment]
Thank you! Working perfectly now!
